submissions/hello_world/DogOrCat/README.txt

```diff
@@ -0,0 +1,10 @@
+This program shows an image of a cat or a dog.
+
+*******
+
+To Run the program, ensure your path leads to the
+DogOrCat folder.
+
+Once you are in the correct directory. type 'python catordog.py'.
+
+*******
```

Comment:
Your README should be a markdown file, and missing the following info:
- List of contributors
- Why you contributed it